package.json

```diff
@@ -7,6 +7,11 @@
   "author": "Bryan Helmig <bryan@zapier.com>",
   "license": "UNLICENSED",
   "main": "zapier.js",
+  "files": [
+      "lib/*.js",
+      "lib/commands/*.js",
+      "lib/utils/*.js"
```

Comment:
sweet! There's just a couple of things we're missing that i'm 99% sure we need:

* zapier.js
* scaffold/

Reply:
* `zapier.js` is specified as `main`, so npm includes it even if we don't specify it in `files`. But on the second thought, it always better to be explicit, so added in 8a1f125.
* Nice catch on `/scaffold/`. Added in a1e9fb4.

Reply:
ah, you're right about main. that's safe to ignore, but doesn't hurt to have.